### PR TITLE
Allow only creating resources compatible with organization in UI

### PR DESCRIPTION
**Pull request by okahilak**
Part of #1893 

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue

## Documentation
- [ ] update changelog if necessary

## Testing
- [x] complex logic is unit tested
- [ ] valuable features are integration / browser / acceptance tested automatically

### CODE REVIEW

File: src/cljs/rems/administration/create_resource.cljs [hunk 1]
```diff
@@ -85,34 +90,58 @@
...
+  (let [licenses @(rf/subscribe [::licenses])]
+    (sort (distinct (mapv :organization licenses)))))
+
 (defn- resource-organization-field []
-  (let [readonly (roles/disallow-setting-organization? (:roles @(rf/subscribe [:identity])))]
-    [text-field context {:keys [:organization]
-                         :label (text :t.administration/organization)
-                         :readonly readonly
-                         :placeholder (text :t.administration/organization-placeholder)}]))
+  (let [organizations (list-organizations)
```
Comment (opqdonut) [inline src/cljs/rems/administration/create_resource.cljs]: hmm I think this is wrong. consider the case where an installation doesn't use resource licenses at all. not sure what the fix would be...

File: src/cljs/rems/administration/create_resource.cljs [hunk 2]
```diff
@@ -85,34 +90,58 @@
...
      [:label {:for licenses-dropdown-id} (text :t.create-resource/licenses-selection)]
-     [dropdown/dropdown
-      {:id licenses-dropdown-id
-       :items available-licenses
-       :item-key :id
-       :item-label #(get-localized-title % language)
-       :item-selected? #(contains? (set selected-licenses) %)
-       :multi? true
-       :on-change #(rf/dispatch [::set-licenses %])}]]))
+     (cond
```
Comment (opqdonut) [inline src/cljs/rems/administration/create_resource.cljs]: two-case cond could as well be an if

File: src/cljs/rems/administration/create_resource.cljs (hunk 1 above)
Comment (okahilak) [reply src/cljs/rems/administration/create_resource.cljs]: After discussion, getting the list of organizations from config now.

File: src/cljs/rems/administration/create_resource.cljs (hunk 2 above)
Comment (okahilak) [reply src/cljs/rems/administration/create_resource.cljs]: Done.

File: src/cljs/rems/administration/create_resource.cljs [hunk 3]
```diff
@@ -16,35 +17,51 @@
...
+
+                        :else
+                        nil)]
      {:db (assoc db
-                 ::form (merge {:licenses #{}}
-                               (when (roles/disallow-setting-organization? roles)
+                 ::form (merge {:licenses []}
+                               (when organization
                                  {:organization organization}))
+                 ::organization-read-only? (not (nil? organization))
```
Comment (opqdonut) [inline src/cljs/rems/administration/create_resource.cljs]: good idea to move the read-only logic here :+1:

**Pull request merged**